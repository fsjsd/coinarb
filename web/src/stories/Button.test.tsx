import * as stories from './Button.stories';
import storybookSnapshotRunner from '../../test/storybookSnapshotRunner';

storybookSnapshotRunner(stories);