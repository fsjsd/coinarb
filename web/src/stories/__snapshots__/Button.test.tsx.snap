// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Large story 1`] = `
<body>
  <div>
    <button
      class="storybook-button storybook-button--large storybook-button--secondary"
      type="button"
    >
      Button
    </button>
  </div>
</body>
`;

exports[`Renders Primary story 1`] = `
<body>
  <div>
    <button
      class="storybook-button storybook-button--medium storybook-button--primary"
      type="button"
    >
      Button
    </button>
  </div>
</body>
`;

exports[`Renders Secondary story 1`] = `
<body>
  <div>
    <button
      class="storybook-button storybook-button--medium storybook-button--secondary"
      type="button"
    >
      Button
    </button>
  </div>
</body>
`;

exports[`Renders Small story 1`] = `
<body>
  <div>
    <button
      class="storybook-button storybook-button--small storybook-button--secondary"
      type="button"
    >
      Button
    </button>
  </div>
</body>
`;
